setup.py

```diff
@@ -55,7 +55,6 @@ def find_version(*file_paths):
         'flask>=0.10.0',
         'lxml>=3.4.4',
         'spyne>=2.12.8',
-        'nose',
```

Comment:
It may take a while for a good odm2api pypi package to be available ... Probably at least a week.